Moli/Core/Screens/UserLists/UserListView.swift

```diff
@@ -42,7 +46,18 @@ struct UserListView: View {
         }
     }
     
-    init(user: UserAPI.UserResponse) {
+    init(startingOn: ListType? ,user: UserAPI.UserResponse) {
+        switch startingOn {
+        case .friends:
+            self.selectedTabIndex = 0
+        case .following:
+            self.selectedTabIndex = 2
+        case .followers:
+            self.selectedTabIndex = 1
+        default:
+            self.selectedTabIndex = 0
+        }
+
```

Comment:
:art: För att slippa `default` case:t, så kan du ha en initial value istället 
```suggestion
    init(startingOn: ListType = .friends, user: UserAPI.UserResponse) {
        switch startingOn {
        case .friends:
            self.selectedTabIndex = 0
        case .following:
            self.selectedTabIndex = 2
        case .followers:
            self.selectedTabIndex = 1
        }
```